draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -973,18 +977,18 @@ most likely believes themselves to be interacting. We'll label this domain the
 "top-level site".
 
 For a document displayed in a top-level browsing context, we can stop here: the
-document's "site for cookies" is the top-level site.
+document's "site for cookies" is top-level site's origin.
 
 For documents which are displayed in nested browsing contexts, we need to audit
 the origins of each of a document's ancestor browsing contexts' active documents
 in order to account for the "multiple-nested scenarios" described in Section 4
-of {{RFC7034}}. A document's "site for cookies" is the top-level site if and
-only if the document and each of its ancestor documents' origins have the same
-registrable domain as the top-level site. Otherwise its "site for cookies" is
-the empty string.
+of {{RFC7034}}. A document's "site for cookies" is the top-level site's origin 
+if and only if the document and each of its ancestor documents' origins have the
+same origin as the top-level site. Otherwise its "site for cookies" is an origin
```

Comment:
We still want to only require the ancestors to share a (scheme, registrable domain) with the top-level site. Requiring the whole chain of ancestors to be same-origin is too strict.

I think this should say something like

> A document's "site for cookies" is the top-level site's origin if and only if the top-level site's origin is same-site with the document's origin and with each of the document's ancestor documents' origins.